packages/magento-product/plugins/StickyAddToCart/StickyProductPageAddToCartRow.tsx

```diff
@@ -0,0 +1,41 @@
+import type { IfConfig, ReactPlugin } from '@graphcommerce/next-config'
+import { Box } from '@mui/material'
+import { useEffect, useRef, useState } from 'react'
+import { createPortal } from 'react-dom'
+import { ProductPageAddToCartRow, StickyAddToCart } from '../../components'
+
+export const component = 'ProductPageAddToCartActionsRow'
+export const exported =
+  '@graphcommerce/magento-product/components/ProductPage/ProductPageAddToCartRow'
+export const ifConfig: IfConfig = 'enableStickyAddToCart'
```

Comment:
This plugin is self-referencing. So the plugin is targeting the current package. Shouldn't we make this a simple if statement somewhere? @carlocarels90  / @mikekeehnen.

Reply:
We could, but we could also make it a separate package right @paales? This would make it a plug and play solution

Reply:
@mikekeehnen if we do this, we don't need the config file I guess? Because it will apply only when you add this package to your project?